transcendence-app/src/config/env.validation.ts

```diff
@@ -49,11 +66,10 @@ export class EnvironmentVariables {
 }
 
 export function validate(config: Record<string, unknown>) {
-  const validatedConfig = plainToClass(EnvironmentVariables, config, {
-    enableImplicitConversion: true,
-  });
+  const validatedConfig = plainToClass(EnvironmentVariables, config);
   const errors = validateSync(validatedConfig, {
     skipMissingProperties: false,
+    forbidUnknownValues: true,
```

Comment:
just a question:
Doesn't the fact that all the attributes in EnvironmentVariables are marked as `!` required, overlap with this forbidUnknownValues?

Reply:
The `!` is the [non-null assertion operator](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-). This operator tells TypeScript that these values can't be null or undefined because they are validated here at runtime.